src/components/storage/CockpitStorageIntegration.scss

```diff
@@ -57,3 +60,25 @@ ul.cockpit-storage-integration-requirements-hint-list {
         color: unset;
     }
 }
+
+// Simulate Backdrop behavior for modals in the iframe
+%iframe-zindex {
+  position: relative;
+  z-index: 10;
+}
+
+iframe {
+  @extend %iframe-zindex;
+}
+
+// Simulate a PF background scrim
+.cockpit-has-modal {
+  @extend %iframe-zindex;
```

Comment:
We're reusing that same SCSS here.

After the two `@extend`s are used, we basically get this CSS:

```css
iframe, .cockpit-has-modal {
  position: relative;
  z-index: 10;
}
```